### PR TITLE
Backend/load user details#145

### PR DESCRIPTION
**Pull request by GD-Dheer**
This API loads user details. 
Returns First, last name. Organization, birthday, and admin status.

### CODE REVIEW

File: Server/src/routes/authenticationRouter.ts [hunk 1]
```diff
@@ -22,6 +23,11 @@ router.post('/login', (request: Request, response: Response, next: NextFunction)
 //TODO: Implement when doing password reset 
 router.get('/resetPassword', (request: Request, response: Response, next: NextFunction) => {
 
-})
+});
+
+router.get('/userDetails', JWTAuthenticator.verifyJWT, async (request: Request, response: Response, next: NextFunction) => {
```
Comment (TiraFire) [inline Server/src/routes/authenticationRouter.ts:28]: would you be able to give me a rundown on how this authenticator functions?

**Review by TiraFire — APPROVED** [review]
Looks good, thank you for the tests as well, ill be looking forward to the explanation

File: Server/src/routes/authenticationRouter.ts (hunk 1 above)
Comment (GD-Dheer) [reply Server/src/routes/authenticationRouter.ts:28]: Yes anytime. We can do it on discord

**Review by Mehrdadmaskull — APPROVED** [review]
LGTM, we'll need to document how to use the JWT on authorizing other parts of the API

**Pull request merged**